edx_rest_framework_extensions/settings.py

```diff
@@ -33,3 +38,25 @@ def get_setting(name):
         return getattr(settings, 'EDX_DRF_EXTENSIONS', {})[name]
     except KeyError:
         return DEFAULT_SETTINGS[name]
+
+
+def get_jwt_issuers():
+    """
+    Retrieves the JWT_ISSUERS list from settings.  If no list is defined, an attempt is made to
+    return the deprecated JWT configuration settings instead.
+    """
+    # If we have a 'JWT_ISSUERS' list defined, return it
+    jwt_issuers = getattr(settings, 'JWT_ISSUERS', None)
```

Comment:
For consistency, `settings` should be `settings.JWT_AUTH`. All of our JWT-related authentication settings are in the `settings.JWT_AUTH` dict.